src/crewai/tools/tool_usage.py

```diff
@@ -244,6 +249,7 @@ def _use(
             tool_calling=calling,
             from_cache=from_cache,
             started_at=started_at,
+            result=result,
```

Comment:
can u add some test to ensure we are emitting `ToolUsageFinishedEvent` with the `result` attributes?